packages/common/mui/src/lib/data-table-cells/icon-cell/icon-cell.tsx

```diff
@@ -11,8 +11,18 @@ export interface IconCellProps {
 export const IconCell = ({ cell: { value } }: IconCellProps): JSX.Element => {
   return (
     <Box sx={{ display: 'flex', alignItems: 'center' }}>
-      <TokenIcon symbol={value || 'unknown'} size="medium" />
-      <TableCell style={{ marginLeft: '10px', lineHeight: 'normal' }}>{value}</TableCell>
+      {value !== 'Total' && (
+        <TokenIcon symbol={value || 'unknown'} size="medium" />
+      )}
+      <TableCell
+        style={{
+          marginLeft: value !== 'Total' ? '10px' : '',
```

Comment:
Can we export "Total" as a constant from this file so that we don't have to rely on a string match?